request-management-api/request_api/models/masterdataimport/receivedmodes.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO public."ReceivedModes"(name, description, isactive) VALUES ('Email', 'Email',  True);commit;
```

Comment:
Going forward, I really want us to discuss a strategy of these SQL files that's more compatible with OpenShift

Ideally they should be part of the Python code as starter data, like queue-management.  This way they can be a part of the CI/CD scripts.  Otherwise we'll have to write some custom bash scripts that just wind up calling these SQL files, which will be more brittle than using the proper SQL alchemy stuff.   

Here's the file queue-management uses to set their starter data

https://github.com/bcgov/queue-management/blob/master/api/manage.py

Reply:
Yes, I am planning to create a setup.py to do this work. Can we treat this as a tech. debt and move ahead? I will open a ticket